### ISSUE TITLE
Hint text doesn't align with different size input

### ISSUE THREAD

**Issue by IPWright83** [ericgio/react-bootstrap-typeahead#501]
### Version

3.4.7

### Steps to reproduce

Set a different height on a `.form-control`. Here is an example https://3scv3.csb.app/

### Expected Behavior

The hint text should line up with the typed text

### Actual Behavior

The hint text is offset 
![image](https://user-images.githubusercontent.com/1374775/68158572-ce468600-ff47-11e9-85bb-0071a3b4f682.png)

**Comment by jamestewartjr** [comment]
We had the same problem. We found that we didn't need the hint so we made it go away.
 ```
.rbt-input-hint {
  display: none !important;
}```

**Comment by ericgio** [comment]
Note that you can control the size of the input [via the `bsSize` prop](http://ericgio.github.io/react-bootstrap-typeahead/#input-size). You can override the `.input-lg` or `.input-sm` selectors if you need a different height.

**Comment by ericgio** [comment]
This should be fixed in v4.0.0-rc.1

**Comment by andresgutgon** [comment]
I fixed with 

```
.rbt-input-hint {
   top: -1px !important;
}
```
